.gitignore

```diff
@@ -36,3 +36,7 @@ common/temp/**
 package-deps.json
 *.build.log
 *.build.error.log
+
+
+#igoner DS store
+.DS_Store
```

Comment:
Looks like one of these was added under `packages/.DS_store`, can you remove it?

Reply:
@archana-maharjan here I was saying that the actual `.DS_Store` files have been added in this PR. You can leave the entry for it in the `.gitignore`.

Reply:
Lol, sorry I misread your statement. Both DS_store removed please!